--- ./lyricviewer.sh~	2005-08-16 23:46:30.000000000 +0300
+++ ./lyricviewer.sh	2005-08-17 00:09:31.000000000 +0200
@@ -27,7 +27,7 @@
 	do
 		#kfmexec "dcop amarok player lyricsByPath" $song | sed -e s'~<br
 		#\/>~~g' -e s'~<\/body>~~' -e s'~<\/html>~~' | tee .lyrics_final
-		kfmexec "dcop amarok player lyricsByPath" $song | html2text	-nobs | tee .lyrics_final
+		kioexec "dcop amarok player lyricsByPath" $song | html2text	-nobs | tee .lyrics_final
 		if [ -n "$(cat .lyrics_final)" ] ; then	
 			kdialog --icon amarok --title "View Lyrics"  --textbox .lyrics_final 225 500
 		else 
